.github/security.md

```diff
@@ -0,0 +1,31 @@
+Thanks for helping make GitHub safe for everyone.
+
+## Security
+
+GitHub takes the security of our software products and services seriously, including all of the open source code repositories managed through our GitHub organizations, such as [GitHub](https://github.com/GitHub).
```

Comment:
I may be reading this wrong, but should this say Primer instead of GitHub? 

```suggestion
GitHub takes the security of our software products and services seriously, including all of the open source code repositories managed through our GitHub organization, such as [Primer](https://github.com/primer).
```